lib/src/apivideo_player_overlay.dart

```diff
@@ -68,6 +68,8 @@ class _ApiVideoPlayerOverlayState extends State<ApiVideoPlayerOverlay>
   bool _isPlaying = false;
   bool _didEnd = false;
 
+  late double widgetWidth;
```

Comment:
Is there a better way to get widget width?
But what if the width is huge and the height small?